srcCpp/Infrastructure_pro.h

```diff
@@ -66,23 +66,34 @@ inline bool PerftestSemaphore_give(PerftestSemaphore *sem)
         PerftestSemaphore_take(sem, PERFTEST_SEMAPHORE_TIMEOUT_INFINITE)
 
 #define PERFTEST_DISCOVERY_TIME_MSEC 1000 // 1 second
+#define ONE_BILLION  1000000000L // 1 billion (US)
+
+#ifndef RTI_PERFTEST_NANO_CLOCK
+  #define PERFT_TIME_UNIT "us"
+#else
+  #define PERFT_TIME_UNIT "ns"
+#endif
```

Comment:
If this is the same that is defined on Infrastructure_micro.h we could move it into Infrastructure_common.h

Reply:
Agreed. Will do.